08LessonEight/homework/exercise-06.html

```diff
@@ -2,9 +2,56 @@
 <html>
 
 <head>
-    <title>Exercise 06</title>
+    <title>Homework Lesson 8</title>
+    <link href="css/exercise-06-style.css" rel="stylesheet" />
 </head>
 
-<body></body>
+<body>
+    <main>
 
-</html>
\ No newline at end of file
+        <!--first section  -->
+        <section>
+            <img src="img/truck.jpg" alt="Truck" title="Truck" />
+            <div>
+                <h2>SAFE & SECURE DELIVERY</h2>
+                <p>
+                    Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+                </p>
+                <div class="read-more">
+                    <a href="#">READ MORE</a>
+                </div>
+            </div>
+        </section>

```

Comment:
everything lines up in your page precisely! great job organizing and commenting your code, christina :D